Jenkinsfile

```diff
@@ -433,6 +433,7 @@ pipeline {
                                         try {
                                             sh "npm run test -- brave_unit_tests ${BUILD_TYPE} --output brave_unit_tests.xml"
                                             xunit([GoogleTest(pattern: "src/brave_unit_tests.xml", deleteOutputFiles: false, failIfNotNew: true, skipNoTestFiles: false, stopProcessingIfError: false)])
+                                            xunit([GoogleTest(pattern: "src/brave_installer_unittests.xml", deleteOutputFiles: false, failIfNotNew: true, skipNoTestFiles: false, stopProcessingIfError: false)])
```

Comment:
cc: @mihaiplesa for a heads up about this new file 😄

Reply:
whoa nice, we're just adding testing of DMG/PKG to the pipeline